commands/transform.go

```diff
@@ -68,9 +71,11 @@ func transformSlackCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	// attachments dir
+	attachmentsRealDir := path.Join(attachmentsDir, attachmentsInternal)
```

Comment:
nit: I'd suggest to use a more descriptive name for this one, like `attachmentsFullDir`

Reply:
Good point. I changed it accordingly.